pkg/extract/types/data/detection/criteria/criterion/criterion.go

```diff
@@ -16,10 +16,24 @@ import (
 
 type Criterion struct {
 	Vulnerable bool                    `json:"vulnerable,omitempty"`
+	FixStatus  *FixStatus              `json:"fix_status,omitempty"`
```

Comment:
What about implementing FixStatus from criterion to criterion/fixstatus, like package and affected?
(I fully understand that there is no particular reason to bring it up, but it is done as a matter of convention.)

Reply:
Will do!

Reply:
done by https://github.com/MaineK00n/vuls-data-update/compare/90d60232325399be00bb0c122653d987a0e0c4bf..6ae43f322f8a32f5046bf256b5c487a08f77af4a